src/integer_mod_q/modulus_polynomial_ring_zq/from.rs

```diff
@@ -72,6 +76,70 @@ impl From<&ModulusPolynomialRingZq> for ModulusPolynomialRingZq {
     }
 }
 
+impl ModulusPolynomialRingZq {
+    /// Outputs a [`ModulusPolynomialRingZq`] of the form `X^n + 1 mod modulus`.
+    ///
+    /// Parameters:
+    /// - `n`: specifies the degree of the modulus polynomial
+    /// - `modulus`: specifies the modulus of the modulus polynomial
+    ///
+    /// Returns a [`ModulusPolynomialRingZq`] of the form `X^n + 1 mod modulus` or
+    /// a [`MathError`] if `modulus <= 1`, `n < 0`, or `n` does not into an [`i64`].
+    ///
+    /// # Examples
+    /// ```
+    /// use qfall_math::integer_mod_q::ModulusPolynomialRingZq;
+    ///
+    /// let poly_mod = ModulusPolynomialRingZq::new_anticyclic(8, 17);
+    /// ```
+    ///
+    /// # Errors and Failures
+    /// - Returns a [`MathError`] of type [`OutOfBounds`](MathError::OutOfBounds) if
+    /// the `n` is negative or it does not fit into an [`i64`].
+    ///
+    /// # Panics ...
+    /// - if the `modulus` is not larger than `1`.
+    pub fn new_anticyclic(
+        n: impl TryInto<i64> + Display,
+        modulus: impl Into<Modulus>,
+    ) -> Result<Self, MathError> {
+        let mut poly = PolyOverZq::from((1, modulus));
+        poly.set_coeff(n, 1)?;
+        Ok(ModulusPolynomialRingZq::from(&poly))
+    }
```

Comment:
This modulus only indirectly defines an anti-cyclic lattice (i.e. using the coefficeint embedding) otherwise this only defines an anti-cyclic ideal.